lib/screens/home.dart

```diff
@@ -243,13 +244,15 @@ class _HomeScreenState extends State<HomeScreen> with TickerProviderStateMixin {
 
                         if (!ttsModel.enabled) {
                           await TextToSpeechPlugin.clear();
+                          FlutterBackgroundService().invoke("stopService");
```

Comment:
Same here, this should be under a more strict flag.

Reply:
I kept this commented in otherwise the CI would fail.